data/config.yml

```diff
@@ -104,6 +104,19 @@ dashboard:
     "Dashboard - Link to Vulnerability: Security rule details":
       selectors:
         - '{} .ins-c-dashboard-data-list a'
+    "Dashboard - Link to Patch: View all systems affected":
+      selectors:
+        - '{}[data-ouia-app-id="dashboard"] .ins-c-dashboard__card--PatchManager a:contains("systems affected")'
```

Comment:
`[data-ouia-app-id="dashboard"]` is not needed in these, this is taken care of by the `scope` above.

```suggestion
        - '{} .ins-c-dashboard__card--PatchManager a:contains("systems affected")'
```